_data/versions.yml

```diff
@@ -8,10 +8,26 @@ v3.0:
     ##### Support for etcdv3
 
     - Calico now stores its data in [etcd version 3](https://coreos.com/blog/etcd3-a-new-etcd.html).
-    
+
+    > **Important**: If using the etcd datastore it is a requirement to use v3,
```

Comment:
"requirement to use v3" probably doesn't give enough context or make it clear what v3 means.  e.g. Could be misunderstood as "you need to be running Calico v3 if you are using etcd", or perhaps "you need to be running etcd v3".

Perhaps something like this captures it better: "If you are upgrading a previous Calico deployment to Calico 3.0 you must follow this one-time migration process as part of the upgrade process.  Please read and make sure you fully understand the upgrade process before attempting to upgrade to Calico 3.0."